README.md

```diff
@@ -1,75 +1,41 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+Project Name : Hello Microverse
```

Comment:
[OPTIONAL]
You followed the general template provided by Microverse 👏 However it's better to include some markdown to improve the readability and make it look better 👍 Kindly check [this link](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) for more information.